test/pkg/integration/apis.go

```diff
@@ -491,12 +491,11 @@ func (c *ComponentAPI) CreateUser(username string, token string) (string, error)
 	}
 	if authId == "" {
 		authId = strconv.FormatInt(time.Now().UnixMilli(), 10)
-		_, err = db.Exec(`INSERT IGNORE INTO d_b_identity (authProviderId, authId, authName, userId, tokens) VALUES (?, ?, ?, ?, ?)`,
```

Comment:
@kylos101 Could you approve please to unblock JB update? 

For context see https://gitpod.slack.com/archives/C032A46PWR0/p1685973787702279

Reply:
@akosyakov do jetbrains tests run with `with-integration-tests=all`? I ask to assert that workspace tests are still working, too.

Reply:
Can be done with another PR? That we don't blcok JB upgade withing for build again?

Reply:
nvm they are failing any more, let's rerun all of them

Reply:
@kylos101 All JB tests are passing (@iQQBot can we get an approve on this PR from IDE team 🙏 ), I found following failed other tests not related to IDEs:
```
2023-06-05T19:22:15.0182695Z [test-server]         --- FAIL: TestStartWorkspace/CreateWorkspace/it_can_run_workspace_tasks (19.49s)
...
2023-06-05T20:36:03.0303764Z [test-workspace]             --- FAIL: TestGitHubContexts/context/should_run_context_tests/github.com/gitpod-io/gitpod-test-repo/tree/integration-test-1 (509.23s)
...
2023-06-05T20:43:33.9819932Z [test-workspace]         --- FAIL: TestRunDocker/docker/it_should_start_a_container (300.44s)
...
2023-06-05T20:50:55.9688839Z [test-workspace]             --- FAIL: TestGitActions/GitActions/it_can_run_git_actions/github.com/gitpod-io/gitpod-test-repo/tree/integration-test/commit_classic (150.57s
```